ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -271,7 +271,7 @@ StatusDialog {
                         }
 
                         StatusListItemTag {
-                            Layout.maximumWidth: 150
+                            Layout.maximumWidth: 300
```

Comment:
I'm not sure about this fix if I read the task. 

From the task I'm not understanding the design needs to be changed by increasing the `pill width` but what we need to prevent  is cutting the _token symbol_. To do that, if we need to keep the same pill size, probably we should change the `elide mode` to elide in the `middle`. But, I'd like to confirm it with design team just to be sure we are understanding that correctly. @John-44 @benjthayer , am I correct?

If so, it's just matter of changing `titleText.elide: Text.ElideMiddle`.

Reply:
My interpretation was to not elide this ever